### PR TITLE
Instructions for loading semantics in HM3D

### PR DESCRIPTION
**Pull request by srama2512**
## Motivation and Context

This PR adds useful information for loading semantic annotations in HM3D.

## How Has This Been Tested

Tested locally.

## Types of changes

- [X] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: DATASETS.md [hunk 1]
```diff
@@ -31,6 +31,75 @@ By default, downloading the data for train/val/example scenes also pulls in the
...
+TEEsavR23oF.basis.glb       TEEsavR23oF.basis.navmesh
+TEEsavR23oF.semantic.glb    TEEsavR23oF.semantic.txt
+```
+Note that there may be more files in `<PATH TO HM3D>/minival/00800-TEEsavR23oF/` if the full HM3D dataset is downloaded. Most importantly, ensure that the `hm3d_annotated_*`, `*.semantic.glb`, and `*.semantic.txt` files are present.
+
+To load semantic annotations in habitat-sim:
+* Enable the semantic sensor
+* Set the `scene_dataset_config_file` configuration variable
+
+A simple example below:
```
Comment (ykarmesh) [inline DATASETS.md:62]: Noob question: Where can a user use this script?

File: DATASETS.md [hunk 2]
```diff
@@ -31,6 +31,75 @@ By default, downloading the data for train/val/example scenes also pulls in the
...
+agent_cfg = habitat_sim.agent.AgentConfiguration()
+agent_cfg.sensor_specifications = [sem_cfg]
+
+sim_cfg = habitat_sim.Configuration(backend_cfg, [agent_cfg])
+sim = habitat_sim.Simulator(sim_cfg)
+```
+
+To view the semantic annotations in the C++ viewer, install the latest `main` branch of habitat-sim and run the following command:
+```
+./build/viewer --dataset '<PATH TO HM3D>/hm3d_annotated_basis.scene_dataset_config.json' TEEsavR23oF
```
Comment (dhruvbatra) [inline DATASETS.md:83]:
```suggestion
# ./build/viewer if compiled locally
habitat-viewer --dataset '<PATH TO HM3D>/hm3d_annotated_basis.scene_dataset_config.json' TEEsavR23oF
```

File: DATASETS.md (hunk 1 above)
Comment (dhruvbatra) [reply DATASETS.md:62]: Same question here. Can we tell the reader where to do this?

**Review by dhruvbatra — APPROVED** [review]

File: DATASETS.md (hunk 1 above)
Comment (srama2512) [reply DATASETS.md:62]: It shouldn't matter as long as <PATH TO HM3D> is set correctly :)

**Review by devendrachaplot — APPROVED** [review]

**Pull request merged**